definitions.yaml

```diff
@@ -284,6 +284,52 @@ definitions:
         type: object
         additionalProperties: true
 
+  V4GetClusterAppsResponse:
+    type: array
+    description: Array of apps
+    items:
+      type: object
+      properties:
+        metadata:
+          type: object
+          properties:
+            name:
+              type: string
+              description: The identifier you set when creating this app
```

Comment:
I don't think we have description in app CRD, it's a desire to install an app, you don't need to describe the app for that

Reply:
nvm, I'm stupid ;)

Reply:
I got confused for a second there too :P